zeroex/order_validator.go

```diff
@@ -363,6 +363,12 @@ type softCancelResponse struct {
 	OrderHashes []common.Hash `json:"orderHashes"`
 }
 
+/**
+ * batchValidateSoftCancelled validates any order specifying the Coordinator contract as the `senderAddress` to ensure
```

Comment:
This style of comment isn't idiomatic and probably won't render correctly on GoDoc or in editors which support inline documentation on hover. Can you change it to:

```go
// batchValidateSoftCancelled validates any order specifying the Coordinator contract as the `senderAddress` to ensure
// that it hasn't been cancelled off-chain (soft cancellation). It does this by looking up the Coordinator server endpoint
// ...
```

Or if you still want to use multi-line comments, remove the extra asterisks:

```go
/*
batchValidateSoftCancelled validates any order specifying the Coordinator contract as the `senderAddress` to ensure
that it hasn't been cancelled off-chain (soft cancellation). It does this by looking up the Coordinator server endpoint
...
*/
```